src/index.js

```diff
@@ -80,6 +82,53 @@ async function main ( args ) {
 	    });
 	}));
 
+	commander
+	.command("install-config <path/to/app-config.yml>")
+	.description("install apps in config.yml")
+	.option('-a, --agent [pubkey]', 'create a new agent for installed app', false)
+	.action( prelude(function ( config ) {
+	    execute_admin(async (AdminAPI) => {
+		let agent_pubkey;
+
+		if ( this.agent === false ) {
+		    agent_pubkey	= await AdminAPI.generate_pubkey();
+		    log.normal("Using Agent (%s) for install", agent_pubkey.toString("base64") );
+		}
+		else {
+		    agent_pubkey	= Buffer.from( this.agent, "base64" );
+		    log.normal("Using Agent (%s) for install", this.agent );
+		}
+
+		if ( config === false ) {
+			throw new Error('Must provide a path to <app-config.yml> to specify apps to install.')
+		}
+		else {
+			// installs list of apps provided in app-config.yml file
+			const apps = yaml.load(fs.readFileSync(config, 'utf8'));
+			for (let app of apps) {
+			  const dna_config = app.dnas.reduce((obj, dnaPath) => {
+				const path = dnaPath.split("/");
+				const buildName = path[path.length - 1];
+				const nick = buildName.split('.dna')[0];
+				obj[nick] = dnaPath;
```

Comment:
Let's change this to be consistent with the install format for specifying the DNA nickname.

- `<...dna.gz filepath>:<nick>`